config/default/webhook_ocp_cainjection_patch.yaml

```diff
@@ -0,0 +1,7 @@
+# This patch adds an annotation to the validation webhook config to tell OpenShift to inject a CA
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: validating-webhook-configuration
```

Comment:
Is this the eventual name of the webhook? If so, it is too generic and might collide with other webhook names and we need to change it

Reply:
The final name is `ssp-operator-update-validating-webhook-configuration`, so we should be good here.